### PR TITLE
test: added integration test for container

### PR DESCRIPTION
**Pull request by athakor**
Towards #11

### CODE REVIEW

File: google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      }
+    }
+  }
+
+  @Test
+  public void getServerConfigTest() {
+    ServerConfig config = client.getServerConfig(PROJECT_ID, ZONE);
+    assertEquals("COS", config.getDefaultImageType());
+    assertEquals(
+        "1.14.7-gke.10",
```
Comment (chingor13) [inline google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java]: This seems like this value depends on the GKE version and is subject to change without any code changes.

File: google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        config.getValidNodeVersions(ServerConfig.VALID_MASTER_VERSIONS_FIELD_NUMBER));
+    assertEquals(
+        "1.12.10-gke.20",
```
Comment (chingor13) [inline google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java]: This seems like this value depends on the GKE version and is subject to change without any code changes.

**Comment by athakor** [comment]
@chingor13 PTAL

File: google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java (hunk 1 above)
Comment (athakor) [reply google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java]: Done. removed check,related to GKE version.

File: google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java (hunk 2 above)
Comment (athakor) [reply google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java]: Done. removed check,related to GKE version.

**Pull request merged**